BackendServer/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "express-4.x-local-example",
+  "version": "0.0.0",
+  "description": "Express 4.x app using Passport for authentication with username and password.",
+  "keywords": [
+    "example",
+    "express",
+    "passport"
+  ],
+  "author": "Jared Hanson",
```

Comment:
Who's Jared?

Reply:
Fixed, started with an open source sample for passport. Since been removed.